packages/gestalt/src/SideNavigation/ItemContent.tsx

```diff
@@ -115,7 +129,7 @@ export default function ItemContent({
 
   const inactiveItemColor = hovered ? 'secondary' : undefined;
   const itemColor = active ? 'selected' : inactiveItemColor;
-  const textColor = active ? 'inverse' : 'default';
+  const textColor = getTextColor({active, disabled})
```

Comment:
I don't think this applies to the label and subtext equally. 

subtext shouldn't be subtle instead of default?